src/library/FOSSBilling/Monolog.php

```diff
@@ -29,22 +29,13 @@ class Monolog implements InjectionAwareInterface
         "event"
     ];
 
-    public function setDi(\Pimple\Container $di): void
```

Comment:
Also just remembered, we need to verify that nothing is going to try to set the DI when calling Monolog now.
Otherwise, we will end up with a fatal exception when it tries to call an undefined function

Reply:
It looks like the only place Monolog is called directly is within the DI, which has been updated. 🙂